### ISSUE TITLE
Wiki plugin - Nicer style in kube console

### ISSUE THREAD

**Issue by davsclaus** [hawtio/hawtio-wiki#19]
The library tab uses wiki, but we should style this nicer. And also the top nav bar has that panel overlapping and whatnot.

![screen shot 2015-04-23 at 10 23 11 am](https://cloud.githubusercontent.com/assets/477100/7293163/6603db8e-e9a2-11e4-8b00-1a71962d86cb.png)

**Comment by gashcrumb** [comment]
Yep, we should actually have the wiki maybe use a custom template for it's nav bar like hawtio-jmx's 'Threads' tab which should help get a consistent look.
